services/s3/src/test/java/software/amazon/awssdk/services/s3/functionaltests/PutObjectWithChecksumTest.java

```diff
@@ -35,6 +35,7 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Disabled;
```

Comment:
Interesting, I seem to remember trying this and checkstyle complaining about it.

Reply:
Checkstyle complains about @Ignore! It doesn't mind @Disabled. That's probably a gap we should fix, but it's convenient right now!